eth/api.go

```diff
@@ -1111,7 +1112,7 @@ func (tx *Tx) UnmarshalJSON(b []byte) (err error) {
 		tx.GasLimit = rpc.NewHexNumber(0)
 	}
 	if tx.GasPrice == nil {
-		tx.GasPrice = rpc.NewHexNumber(defaultGasPrice)
+		tx.GasPrice = rpc.NewHexNumber(50000000000)
```

Comment:
I'm still not entirely sure what this is good for, or if it's actually good here. Shouldn't we use the gas price oracle here too?